### PR TITLE
[166720110] User should be able to update an article

### PR DESCRIPTION
**Pull request by MCFrank16**
#### Usecase
[usecase link](https://docs.google.com/document/d/1bxlTFgu7yX-UjSmF4StTy3a6Zt7Vb2DlTHT8dqfEO2g/edit)
#### What does this PR do?

this PR shows how the authors will be able to update his/her article.

#### Description of Task to be completed?

Have the following endpoint working: 
PUT `` /api/articles/:slug``: this endpoint will allow an author who composed it to be able to update it.

#### How should this be manually tested?
* Clone the repo to  local machine.
* Run ``npm install``   to install the dependencies.
* Set up the   ``.env`` as instructed by the   ``.env-example``
* Run  ``npm  run dev``.
* PUT registration data to  ``/api/articles/:slug``.

#### Any background context you want to provide?
this feature is still waiting for this [PT](https://www.pivotaltracker.com/story/show/166240826) to be finished so keep in mind that the data used are counterfeit.
#### What are the relevant pivotal tracker stories?
[166720110](https://www.pivotaltracker.com/story/show/166720110)

### CODE REVIEW

**Comment by nkpremices** [comment]
Please change the title of this PR
Follow best practices

**Review by nkpremices — APPROVED** [review]
Thank you for applying my previous changes @MCFrank16

**Comment by truestbyheart** [comment]
Resolve the conflicts and change the title

File: src/api/controllers/articleController.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      tagList: updateContent.tagList,
+      category: updateContent.category,
+    }, {
+      where: {
+        slug,
+      }
+    });
+    return res.status(statuses.OK).json({
+      status: statuses.OK,
+      message: 'Your Article is up-to-date now, Thanks',
```
Comment (truestbyheart) [inline src/api/controllers/articleController.js:55]: why not ``successfully updated``?

File: src/api/migrations/20190624184423-create-sequelize-meta.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+    
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('SequelizeMeta', {
+    name: {
+      type: Sequelize.STRING
+    }
+  }),
+  down: (queryInterface, Sequelize) => queryInterface.dropTable('SequelizeMeta')
+};
```
Comment (truestbyheart) [inline src/api/migrations/20190624184423-create-sequelize-meta.js]: add a new line

File: src/api/seeders/20190624184857-Article.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        userId: 1,
+        tagList: ['UUID', 'express', 'sequelize'],
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+    ],
+    {}),
+
+  down: (queryInterface, Sequelize) => queryInterface.bulkDelete('Articles', null, {})
+};
```
Comment (truestbyheart) [inline src/api/seeders/20190624184857-Article.js]: add new line

**Review by truestbyheart — CHANGES_REQUESTED** [review]
@MCFrank16 please kindly work on the feedback

**Pull request closed without merging**